src/main/java/seedu/address/logic/parser/inventory/InventoryAddIngredientCommandParser.java

```diff
@@ -14,8 +30,53 @@
      * and returns a InventoryAddCommand object for execution.
      * @throws ParseException if the user input does not conform the expected format
      */
-    public InventoryCommand parse(String args) throws ParseException {
-        throw new ParseException("Not implemented yet");
+    public InventoryAddIngredientCommand parse(String args) throws ParseException {
+        requireNonNull(args);
+
+        if (containsIngredient(args)) {
+            return parseAddIngredient(args);
+        } else {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, RecipeAddCommand.MESSAGE_USAGE));
+        }
```

Comment:
This part isn't needed for inventory. We needed it for the recipe because we can't tell between adding ingredients and steps just from the command word.

Reply:
ok, will adjust accordingly, thanks!